authenticate-me/backend/routes/groups.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express');
+const { requireAuth } = require('../utils/auth');
+const { Group, Member, User, Image, sequelize } = require('../db/models');
+
+
+const router = express.Router();
+
+//Get Details of a Group from an id
+router.get('/:groupId', async(req, res) => {
+    const { groupId } = req.params;
+
+    const currentGroup = await Group.findByPk(groupId, {
+        include: [
+            {
+                model: Image,
+                attributes: []
+            },
+            {
+                model: Member,
+                attributes: []
+            },
+            {
+                model:User //can't figure out how to alias as 'Organizer'
```

Comment:
as: "Organizer", you will also need to add the alias to the association in your group model

Reply:
ok yeah that's the only solution I had found, so I'll update that now